blocks/jobber/editor.scss

```diff
@@ -0,0 +1,7 @@
+/**
+ * The following styles get applied inside the editor only.
+ */
+
+.wp-block-create-block-jobber {
```

Comment:
I don't see this class anywhere so I don't think this CSS gets applied. That may be fine but if it's not something we're using, we can get rid of it